### PR TITLE
docs: update openid-connect attributes description and sync CN doc attributes.

### PR DESCRIPTION
**Pull request by hf400159**
### Description
update openid-connect attributes description and add CN doc attributes.
add `token_endpoint_auth_method` attributes.

Fixes # (issue)

### Checklist

- [x] I have explained the need for this PR and the problem it solves
- [ ] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [ ] I have updated the documentation to reflect this change
- [x] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

File: docs/zh/latest/plugins/openid-connect.md [hunk 1]
```diff
@@ -47,13 +47,15 @@ description: 本文介绍了关于 Apache APISIX `openid-connect` 插件的基
...
 | ssl_verify                           | boolean | 否     | false                 | [true, false] | 当设置为 `true` 时，验证身份提供者的 SSL 证书。                                                    |
 | introspection_endpoint               | string  | 否     |                       |               | 身份服务器的令牌验证端点的 URL。                                                                   |
-| introspection_endpoint_auth_method   | string  | 否     | "client_secret_basic" |               | 令牌自省的认证方法名称。                                                                           |
+| introspection_endpoint_auth_method   | string  | 否     |                       |               | 令牌自省的认证方法名称，如果未指定，则默认使用 well_know 中的第一个值。                            |
 | public_key                           | string  | 否     |                       |               | 验证令牌的公钥。                                                                                  |
+| use_jwks                             | boolean | 否     |                       |               | 当设置为 `true` 时，则会使用身份认证服务器的 JWKS 端点来验证令牌。                                      |
```
Comment (juzhiyuan) [inline docs/zh/latest/plugins/openid-connect.md:52]: Hi, could you please add the `default` value?

File: docs/en/latest/plugins/openid-connect.md [hunk 2]
```diff
@@ -33,29 +33,29 @@ The `openid-connect` Plugin provides authentication and introspection capability
...
+| redirect_uri                         | string  | False    | "ngx.var.request_uri" |              | URI to which the identity provider redirects back to.                                                                    |
+| timeout                              | integer | False    | 3                     | [1,...]      | Request timeout time in seconds.                                                                                         |
+| ssl_verify                           | boolean | False    | false                 |              | When set to true, verifies the identity provider's SSL certificates.                                                     |
+| introspection_endpoint               | string  | False    |                       |              | URL of the token verification endpoint of the identity server.                                                           |
+| introspection_endpoint_auth_method   | string  | False    |                       |              | Authentication method name for token introspection, If not specified, the default will use the first value in well_know. |
```
Comment (juzhiyuan) [inline docs/en/latest/plugins/openid-connect.md:50]: I'm not clear, cc @starsz to have a check 🙏

Comment (starsz) [reply docs/en/latest/plugins/openid-connect.md:50]: Ok.Let me have a check.

Comment (starsz) [reply docs/en/latest/plugins/openid-connect.md:50]: After digging it into.I found that:

1. The default value of `introspection_endpoint_auth_method` is "client_secret_basic", sorry to @hf400159 .
2. And the `lua-resty-openidc` will use another conf called "token_endpoint_auth_method".It doesn't introduce in the docs, but it can work well in "openid-connection" plugin.

The default value of "token_endpoint_auth_method" will fetch the first supported method specified by the OP.
refer https://github.com/zmartzone/lua-resty-openidc/blob/master/lib/resty/openidc.lua#L677

So in conclusion, I think we should keep the original doc, and add the new filed of "token_endpoint_auth_method" to the docs.

Comment (hf400159) [reply docs/en/latest/plugins/openid-connect.md]: @starsz fixed. PTAL 😄

**Review by starsz — APPROVED** [review]

**Review by juzhiyuan — APPROVED** [review]

**Review by SylviaBABY — APPROVED** [review]
